MAUI/Scheduler/month-view.md

```diff
@@ -55,6 +55,33 @@ N>
 * [View scheduler appointment sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/GettingStarted)
 * [View business object sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/BusinessObject)
 
+## Appointment indicator count
+
+The scheduler month view appointment indicator count allows you to customize the count of the appointment indicator by using `AppointmentIndicatorCount` property of [MonthView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerMonthView.html). By default, `AppointmentIndicatorCount` is set to 5.
```

Comment:
The scheduler month view appointment indicator count allows you to customize the count of the appointment indicator by using the `AppointmentIndicatorCount` property of the [MonthView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerMonthView.html). By default, the `AppointmentIndicatorCount` is set to 5.

Reply:
I have made the changes